packages/manager/modules/phone-prefix/README.md

```diff
@@ -0,0 +1,55 @@
+# account-helpers
+
+> Provides a list of telephone code
+
+## Install
+
+```sh
+yarn add @ovh-ux/manager-phone-prefix
+```
+
+## Imports data
+
+### Use telephony code list
+
+Provides data to allow user to select a right telephone code
+
+#### Usage 
+
+```js
+import PHONE_PREFIX from '@ovh-ux/manager-phone-prefix';
+```
+
+## Build
+
+```sh
+# Build in production mode
+yarn start
+```
+
+## Development
+
+If you want to contribute to the project, follow theses instructions:
+
+Foremost, you should launch a global installation at the root folder of this repository:
+
+```sh
+yarn install
+```
+
+Then you just have to start the project in development mode. For this, two choices are possible according to your needs:
+
+```sh
+# Build the `manager-ng-layout-helpers` workspace and all the nested workspaces in development mode and watch only `manager-ng-layout-helpers` workspace
```

Comment:
```suggestion
# Build the `manager-phone-prefix` workspace and all the nested workspaces in development mode and watch only `manager-phone-prefix` workspace
```

Reply:
`manager-phone-prefix` instead of `phone-prefix`.

Reply:
Done